sklearn/utils/_estimator_html_repr.py

```diff
@@ -402,14 +256,20 @@ def estimator_html_repr(estimator):
             " HTML representation is unable to render, please try loading this page"
             " with nbviewer.org."
         )
-        out.write(
-            f"<style>{style_with_id}</style>"
-            f'<div id="{container_id}" class="sk-top-container">'
-            '<div class="sk-text-repr-fallback">'
-            f"<pre>{html.escape(estimator_str)}</pre><b>{fallback_msg}</b>"
-            "</div>"
-            '<div class="sk-container" hidden>'
-        )
+
+        with open(Path(Path(__file__).parent, "base_template.html")) as fp:
```

Comment:
Can we revert this part back to the old code (right above on the github diff view), as my templating idea did not work out.